### PR TITLE
fix(sdk-core): send vss share to wp

### PR DESCRIPTION
**Pull request by olibeyene**
Ticket: BG-61036

### CODE REVIEW

**Comment by pranavjain97** [comment]
How do we know that vss signing is fixed?

**Review by pranavjain97 — APPROVED** [review]

File: modules/sdk-core/src/bitgo/tss/eddsa/eddsa.ts [hunk 1]
```diff
@@ -200,8 +198,7 @@ export async function offerUserToBitgoRShare(
   const signatureShare: SignatureShareRecord = {
     from: SignatureShareType.USER,
     to: SignatureShareType.BITGO,
-    // TODO(BG-61037): Fix signing with VSS
-    share: /* rShare.v + */ rShare.r + rShare.R,
+    share: rShare.r + rShare.R + rShare.v,
```
Comment (pengyuc-bitgo) [inline modules/sdk-core/src/bitgo/tss/eddsa/eddsa.ts:201]: Why are we putting them all in one string?

Comment (olibeyene) [reply modules/sdk-core/src/bitgo/tss/eddsa/eddsa.ts:201]: this the pattern how tss exchanges shares with WP <> sdk. shares are contacted before sending to wp

**Review by jicli — APPROVED** [review]

File: modules/sdk-core/src/bitgo/tss/eddsa/eddsa.ts (hunk 1 above)
Comment (jicli) [reply modules/sdk-core/src/bitgo/tss/eddsa/eddsa.ts:201]: would hope we had delimiter instead of counting bytes

**Review by pengyuc-bitgo — APPROVED** [review]

**Review by sachushaji — COMMENTED** [review]
Can you add a breaking changes mentioned in the commit message as we are including v value now which prior to the change was not passed. Check [here](https://www.conventionalcommits.org/en/v1.0.0/#:~:text=in%20Semantic%20Versioning\).-,BREAKING,-CHANGE%3A%20a)

**Comment by olibeyene** [comment]
Closing this pr since [pr 3285 handles eddsa vss share](https://github.com/BitGo/BitGoJS/pull/3285 )

**Pull request closed without merging**